provisions/hosts.sample

```diff
@@ -27,9 +27,14 @@ cccp_index_repo=https://github.com/centos/container-index.git
 
 ## dev environment options, update following to true to configure NFS
 setup_nfs=False
+
 # replace scanner_worker below with its FQDN / IP
 #test_nfs_share=scanner_worker:/nfsshare
 
+# for distinguishing emails from production or devcloud
+# keep it True for production and False for non production
+production=True
```

Comment:
Debatable or petty but, when we talked about this, I was suggesting to use something like `test=False`.